projects/plugins/boost/app/lib/Environment_Change_Detector.php

```diff
@@ -70,7 +70,7 @@ public function handle_plugin_change() {
 	 * @param bool   $is_major_change Whether the change is major.
 	 */
 	public function do_action( $is_major_change, $change_type ) {
-		do_action( 'handle_environment_change', $is_major_change, $change_type );
+		do_action( 'jetpack_boost_critical_css_environment_changed', $is_major_change, $change_type );
```

Comment:
I can see there's still an `add_action` call for `handle_environment_change` in `class-speed-score.php`. Does that need updating too?

Reply:
Nice catch. I have updated the hook. But, also kept the `handle_environment_cange` hook in case Jetpack get's released before Boost with the new version of the package.